src/Microsoft.Extensions.Options/IConfigureNamedOptions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Options
+{
+
+    /// <summary>
+    /// Represents something that configures the TOptions type.
+    /// </summary>
+    /// <typeparam name="TOptions"></typeparam>
+    public interface IConfigureNamedOptions<in TOptions> where TOptions : class
```

Comment:
Do you have a plans in future to merge old and new interfaces together? IMHO, if I want to inject `MyOptions` into `MyConsumer1` with one additional configuration **with name** and inject into all other classes with default configuration then I will need to duplicate configurations for old and new interfaces.